app/src/main/java/com/sdp13epfl2021/projmag/activities/ProjectCreationActivity.kt

```diff
@@ -147,7 +169,7 @@ class Form : AppCompatActivity() {
             isTaken = false,
             description = getTextFromEditText(R.id.form_project_description),
             assigned = listOf(),
-            tags = listOf("Default-tag")
+            tags = listTags.toList()
```

Comment:
It's kinda strange that it will not crash if listTags is not initialized in Kotlin (~= null.toList()).